src/nodes/materials/MeshBasicNodeMaterial.js

```diff
@@ -16,7 +16,7 @@ class MeshBasicNodeMaterial extends NodeMaterial {
 		this.isMeshBasicNodeMaterial = true;
 
 		this.lights = true;
-		//this.normals = false; @TODO: normals usage by context
```

Comment:
> normals usage by context

I could not find reference of normal usage over the context object. Is this TODO still relevant?

Reply:
The reason was to remove some possible duplicate operations in `normals` that might occur in this configuration.

Reply:
If you can share an example where  duplicate operations might occur, I would be curious to debug this situation 🤓 .

Otherwise I think we should ensure the correct feature set of the materials first before we apply optimizations. Right now, you could apply a normal map to `MeshBasicMaterial` and it would affect the environment map which isn't ideal.

Reply:
> If you can share an example where duplicate operations might occur, I would be curious to debug this situation

I think it might take a while for me to check this since it would be easier for me to do the tests again than to find the files.

> you could apply a normal map to MeshBasicMaterial and it would affect the environment map sampling which isn't ideal.

For this case I would suggest create a `MeshBasicNodeMaterial.setupNormal()` for modify this default behavior.

Reply:
Okay, there is now an overwritten version of `setupNormal()` in `MeshBasicNodeMaterial` next to a comment that refers to this PR.